umwdevlogs/F1stP0st.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Kitting is a Science
+publishDate: 2022-08-05
+excerpt: In case you were curious
+author: Ender
+draft: false
+tags: [Meaningless, Existensialism, art,]
+---
+
+# New art
+
+Check out this great fact I learned on youtube! 
```

Comment:
Nitpick: should we care about proper editing for these posts? e.g. capitalizing YouTube?

Reply:
Aight. did a redo thingy mabob